metrics-reporter-prometheus/README.md

```diff
@@ -1,5 +1,21 @@
 # metrics-reporter-prometheus
 
+This plugin exposes metrics to be scrapeable by prometheus.
+
+It primarily converts the internally used Dropbox metrics one to one with a few
+notable exceptions:
+
+For any metric that has an ID embedded in its metric name, the id is removed
+from the metric name and added as label `id` to prevent unnecessary
+proliferation of metrics and make them easier to compare and/or aggregate.
+
+For streams and stream rules additional information is provided as labels:
+
+* `stream-title`: title of stream (stream and stream rule)
+* `rule-type`: Type of stream rule, e.g. `regexp`
```

Comment:
aren't stream rules superseded by streams making this not super useful?

Reply:
As long as the dropbox metric exposes this information the converter converts it to a label.